src/WiFiST.cpp

```diff
@@ -215,6 +215,17 @@ int WiFiClass::begin(char* ssid, const char *passphrase)
   return WL_CONNECT_FAILED;
 }
 
+/*
+* @brief  Disconnect the WiFi shield from the current network
+* @param  none
+* @retval none
+*/
+void WiFiClass::disconnect(void)
+{
```

Comment:
What happen if not connect before?

Reply:
You can call `disconnect()` before `begin()` without trouble.
But I added a control on the wifi status to be sure the init of the device is done at least once.